docs/guides/real-time/subscriptions/graphql.md

```diff
@@ -0,0 +1,80 @@
+---
+contributors: Esther Agbaje
+description: "Learn how to get started with Directus' GraphQL subscriptons."
+---
+
+# GraphQL Subscriptions
+
+GraphQL subscriptions provide live updates that are delivered in real-time whenever an item is created, updated or
+deleted in your collection.
+
+This guide assumes you already know [how to establish and authenticate](/guides/real-time/getting-started/graphql) over
+a GraphQL connection.
+
+## Subscribe To Changes In A Collection
+
+Send the following query, `<collection>_mutated` over your WebSocket connection to subscribe to changes. If you want to
+subscribe to a `messages` collection, the query would look like this:
+
+```graphql
+subscription {
+	messages_mutated {
+		id
+		text
+	}
+}
+```
+
+In return, this query will subscribe to changes in the messages collection and return the `id` and `text` fields of the
+message added.
+
+Refer to the [Fields Query Parameter](/reference/query.html#fields) docs for more information on specifying what data
+should be returned.
+
+## Handling Collection Changes
+
+When a change happens to an item in a collection with an active subscription, it will emit a message
+
+```json
+{
+	"messages_mutated": {
+		"id": "1",
+		"text": "Hello world!",
+		"_event": "create"
```

Comment:
Is this up to graphql spec @br41nslug?

Reply:
I dont think its being against spec, but probably not a best practice. Simply calling it `event` made me worry about name collision with an actual field called `event`

Reply:
I agree that `_event` would be less likely for collision.
I believe this should be within specs as it only mentioned that fields should not be prefixed with "__" two underscores.
Ref: https://spec.graphql.org/June2018/#sec-Reserved-Names